cli/create.go

```diff
@@ -100,6 +101,25 @@ func create(containerID, bundlePath, console, pidFilePath string, detach bool,
 		return err
 	}
 
+	if runtimeConfig.FactoryConfig.Template {
```

Comment:
@bergwolf Can we handle this in virtcontainers `CreateSandbox` itself rather than doing this in the cli?
 All these implementation details about the factory could be handled by virtcontainers itself keeping the api simple.

Reply:
kata CLI is the one reading the configuration files. And for frakti, unlike kata cli, we would only initialize the vm factory for only once, instead of doing it for every `CreateSandbox`. That is the reason factory config is not pushed into `CreateSandbox`.